cloud/logger/src/index.ts

```diff
@@ -0,0 +1,77 @@
+import Logger from './Logger';
+import {LOG_LEVEL} from './Logger';
+
+/**
+ * Raincatcher logger implementation, can take a different logger at construction time.
+ */
+export default class LoggerImpl implements Logger {
+  private logger: any;
```

Comment:
What I mean is that while this looks like a nice generic implementation, it doesn't look like it'll work properly with any library, not even the native `console`, so we probably can't give up having specific implementations instead.